### DISCUSSION TITLE
pylon permission

### DISCUSSION THREAD

**Discussion by ApocalypsezZ** [Pryaxis/TShock#2032] [Q&A] [answered]
"You do not have permission to teleport with pylons."
I couldn't use the portal even if I was an admin, or even owner, and I tried to find out that only superadmin could use it. I would like to ask which permission it is. This should be available to all. I would like to add this permission to the VIP group...

**Comment by Someone5641** [comment, marked as answer]
/group addperm VIP tshock.tp.pylon (replace VIP with the name of your vip group)

**Comment by LIMCEND** [comment]
How do I remove a group prefix!?

**Reply by sgkoishi** [reply]
Set it to empty. You may want to use double quote (`""`) to represent it.
